frontend/playground-frontend/src/components/Footer/Footer.tsx

```diff
@@ -5,29 +5,29 @@ import React from "react";
 
 const Footer = () => {
   return (
-    <div id="footer">
+    <div id="footer" data-testid="footer">
       <IconContext.Provider
         value={{
           color: COLORS.dark_blue,
           size: "2.0rem",
         }}
       >
-        <div className="footer-element" id="footer-socials">
-          <a className="foot-element" href={URLs.linkedin}>
+        <div className="footer-element" id="footer-socials" data-testid="footer-socials">
```

Comment:
`data-testid` is a helpful field you can pass into a div to make it easily queryable when it comes to unit testing on the frontend. This field should be unique to the particular div within the confines of the given file